README.md

```diff
@@ -40,40 +40,150 @@ After this, a record for ID `doms.radio:albert-einstein.xml` should be delivered
 
 ## Test with Solr
 
-### Setup Solr
+### Solr fundamentals
+
+In DS, Solr operates with versioned collections. Upon first start or if a full reindex is needed, a timestamped collection is created. Access to the collection is done through two aliases:
+
+* **ds** is used for searching
+* **ds-write** is used for updating
+
+Normally the two aliases will point to the same timestamped collection, but if a full index rebuild is underway, **ds** will point to the complete old collection while **ds-write** will point to the new collection being created. When the full index rebuild has finished, **ds** will be switched to the new collection.
+
+This method ensures that there are no disruptions to Solr search. It is used in production at the Royal Danish Library and also on devel + stage for consistency.
+
+The how-to below expects the version of the Solr configuration to be specified in `schema.xml` as a special field looking like this:
+```xml
+ <field name="_ds_1.0.0_" type="string" indexed="false" stored="false"/>
+```
+If the schema is changed, the version must be updated.
+
```

Comment:
Maybe add a comment on what kind of versioning we are using here? I guess semantic versioning.